model/testing/src/icon4py/model/testing/datatest_fixtures.py

```diff
@@ -189,6 +189,73 @@ def savepoint_velocity_init(data_provider, step_date_init, istep_init, substep_i
     )
 
 
+@pytest.fixture
+def savepoint_velocity_1_7_compute_edge_diagnostics_for_velocity_advection_init(
+    data_provider, step_date_init, istep_init, substep_init
+):  # F811
+    """
+    Load data from ICON savepoint at start of velocity_advection module for combined stencils 1 to 7.
+
+     metadata to select a unique savepoint:
+    - date: <iso_string> of the simulation timestep
+    - istep: one of 1 ~ predictor, 2 ~ corrector of dycore integration scheme
+    - jstep: step count since last boundary interpolation (ranges from 0 to 2*ndyn_substeps-1)
```

Comment:
there is no more `jstep`. rather there is the substep now which is the dynamics substep from 1 to `ndyn_substep`